README.md

```diff
@@ -14,18 +14,21 @@ Learn more about Antenna in [What Antenna Does](antenna-documentation/src/site/m
 
 Please note that some dependencies of SW360antenna are only available for Java 8. So you need to use Java 8 to build the project.
 
-If you want to build antenna without the p2 parts, use
+If you want to build antenna just use maven, e.g.
```

Comment:
Nit: While at it, should we capitalize "Antenna" and "Maven"? "mvn", however, of course needs to stay lower-case.